doc/troubleshooting/precision.md

```diff
@@ -0,0 +1,73 @@
+# Why does a model have low precision?
+
+Many phenomena are caused by model accuracy.
+For example, during simulations, temperatures explode, structures fall apart, and atoms are lost.
+One can [test the model](../test/test.md) to confirm whether the model has the enough accuracy.
+
+There are many reasons for a low-quality model.
+Some common reasons are listed below.
+
+## Data
+
+### Data units and signs
+
+The unit of training data should follow what is listed in [data section](../data/system.md).
+Usually, the package to calculate the training data has different units from those of the DeePMD-kit.
+It is noted that some software label the energy gradient as forces, instead of the negative energy gradient.
+It is neccessary to check them carefully to avoid inconsistent data.
+
+### SCF coverage and data accuracy
+
+The accuracy of models will not exceed the accuracy of training data, so the training data should reach enough accuracy.
+On one hand, SCF should converge to a suitable threshold for all points in the training data.
+Sometimes, QM software may generate unstable outliers, which should be removed.
+On the other hand, the data should be extracted with enough digits and stored with the proper precision.
+Large energies may have low precision when they are stored as the single-precision floating-point format (FP32).
+
+### Enough data
+
+If the model performs good on the training data, but has bad accuracy on another data, this means some data space is not covered by the training data.
+It can be validated by evaluting the [model deviation](../test/model-deviation.md) with multiple models.
+If the model deviation of these data is high for some data, try to collect more data using [DP-GEN](../third-party/out-of-deepmd-kit.md#dp-gen).
+
+### Values of data
+
+One should be aware that the errors of some data is also affected by the absolute values of this data.
+Stable structures tend to be more precise than unstable structures because unstable structures may have larger forces.
+This is the expected behavior and does not mean that the precision of the model is low.
```

Comment:
Error will be introduced in the Projector augmented wave (PAW) dft calculations when the atoms are very close due to the overlap of pseudo-potentials.